pyproject.toml

```diff
@@ -115,3 +122,16 @@ known_vellum = ["vellum", "vellum_cli", "vellum_ee"]
 default_section = "THIRDPARTY"
 sections = ["FUTURE", "TESTING", "STDLIB", "TYPING", "THIRDPARTY", "VELLUM", "FIRSTPARTY", "LOCALFOLDER"]
 no_lines_before = ["STDLIB", "TYPING"]
+
+[tool.black]
+line-length = 120
+target-version = [
+    "py38",
+]
+include = ".*\\.py$"
+exclude = "(.git|venv|src/vellum/client|tests/client|scripts)"
+
+[tool.flake8]
+max-line-length = 120
+exclude = ".git,.venv,ee/codegen/node_modules,ee/codegen_integration/fixtures,scripts,src/vellum/__init__.py,src/vellum/client,src/vellum/core,src/vellum/environment.py,src/vellum/errors,src/vellum/resources,src/vellum/types,tests/client"
```

Comment:
Am highlighting `ee/codegen_integration/fixtures`, meaning we codegen code that fails our linter today (mostly strings that are too long for the file width)